src/main/java/Hubert.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
+
+public class Hubert {
+
+    private static int markIndex(String line, int num) {
+        String markIndexChar = line.substring(num, line.length());
```

Comment:
Very good description of the variable, easy to understand! Good use of camelCase as well!